cli/cmd/addon_download.go

```diff
@@ -73,13 +72,9 @@ func runAddonDownload(ctx context.Context, cmd *cobra.Command, args []string) er
 		return fmt.Errorf("invalid addon ref %q: version must start with 'v'", args[0])
 	}
 
-	c, err := cloudquery_api.NewClientWithResponses(managedplugin.APIBaseURL(),
-		cloudquery_api.WithRequestEditorFn(func(ctx context.Context, req *http.Request) error {
-			req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
-			return nil
-		}))
+	c, err := api.NewClient(getEnvOrDefault(envAPIURL, defaultAPIURL), token.Value)
```

Comment:
why not refactor api url envs as well?

Reply:
Ah, just remembered, because of https://github.com/cloudquery/cloudquery/blob/085548ac922f4c0adee8a024e0f09ea4d9e99ab1/cli/internal/team/team.go#L16 that passes only a URL, so needs another refactoring

Reply:
I'll do it in a separate PR to reduce the diff with this one